tutorials/android-firebase-vision/index.md

```diff
@@ -0,0 +1,215 @@
+
+# Adding Computer Vision to your Android App
+
+Recently, Sara Robinson authored an excellent post explaining one approach
+to [Adding Computer Vision to your App](https://medium.com/@srobtweets/adding-computer-vision-to-your-ios-app-66d6f540cdd2).
+In the post, she introduces the following pattern:
+
+* Use Firebase Auth native client to upload to Firebase Storage
+* Use Cloud Functions to access Vision API and store the result in Firestore
+* Access Cloud Firestore from device to retrieve the API Result
+
+In her application, she is using an iPhone X as her device and as I'm a Pixel 2
```

Comment:
Sara uses iPhone X in her application, and as a Pixel 2 owner, I felt like I was missing out.